dynamic_functions/constantsFunction.py

```diff
@@ -0,0 +1,14 @@
+def string(session_id, string):
+    return str(string)
+
+
+def integer(session_id, integer):
+    return int(integer)
+
+
+def float(session_id, float):
+    return float(float)
+
+
+def boolean(session_id, boolean):
+    return bool(boolean)
```

Comment:
why?

Reply:
These are the function that are actually called by the action assembler when evaluating the dynamic values.

Reply:
aight